packages/cursorless-engine/src/processTargets/modifiers/InstanceStage.ts

```diff
@@ -45,6 +47,7 @@ export class InstanceStage implements ModifierStage {
     }
   }
 
+  // TODO: async I don't know how to deal with that
```

Comment:
@pokey do you know how to deal with that? it is a promise of iterator

![image](https://github.com/cursorless-dev/cursorless/assets/387346/4a4c29f0-8a40-4691-94ff-173313c5dad5)